tests/integration/test_fim/test_registry/test_registry_wildcards/test_registry_wildcards.py

```diff
@@ -0,0 +1,290 @@
+    '''
+    description: Check the behavior of FIM when using wildcards to configure the path of registry keys, and validate
+                 when values are created inside a monitored key, creation, modification and deletion is detected
+                 correctly.
+
+    wazuh_min_version: 4.5.0
+
+    test_phases:
+        - setup:
+            - Set wazuh configuration.
+            - Clean logs files and restart wazuh to apply the configuration.
+        - test:
+            - Check that one or more keys are detected when the configured wildcard is expanded
+            - Create a registry_value inside the first monitored key and check
+            - Wait for scan and check registry_value has been detected as 'added'
+            - Modify the registry_value
+            - Wait for scan and check registry_value has been detected as 'modified'
+            - Delete the registry_value
+            - Wait for scan and check registry_value has been detected as 'deleted'
+        - teardown:
+            - Restore configuration
+            - Stop wazuh
+
+    tier: 1
+
+    parameters:
+        - configuration:
+            type: dict
+            brief: Configuration values to apply to agent.
+        - metadata:
+            type: dict
+            brief: Test case data.
+        - set_wazuh_configuration:
+            type: fixture
+            brief: Set wazuh's configuration file.
+        - configure_local_internal_options_function:
+            type: fixture
+            brief: Set local_internal_options configuration.
+        - restart_syscheck_function:
+            type: fixture
+            brief: restart syscheckd daemon, and truncate the logs.
+        - wait_syscheck_start:
+            type: fixture
+            brief: check that the starting fim scan is detected.
+
+    assertions:
+        - One or more keys have been configured after wildcard expansion
+        - Assert 'registry_value added' event has been detected
+        - Assert 'registry_value modified' event has been detected
+        - Assert 'registry_value deleted' event has been detected
+
+    input_description:
+        - The file 'configuration_registry_wildcards.yaml' contains the configuration template for the test.
+        - The file 'cases_registry_value_wildcards.yaml' contains test case descriptions, configuration values and
+          metadata for each case.
+
+    expected_output:
+        - r".*Expanding entry '.*' to '(.*)' to monitor FIM events."
+        - r".*Sending FIM event: (.+)$" - For 'registry_value' attributes.type and 'added/modified/deleted' type.
+    tags:
+        - scheduled
+    '''
+
+    monitored_keys = get_messages(generate_monitoring_callback(CB_FIM_WILDCARD_EXPANDING), timeout=T_10)
+    assert monitored_keys != [], f"Did not receive expected '{CB_FIM_WILDCARD_EXPANDING}' events"
+
+    subkey = monitored_keys[0].replace(f"{WINDOWS_HKEY_LOCAL_MACHINE}\\", "")
+    subkey = subkey+f"\\{key_name}"
```

Comment:
```suggestion
    subkey = subkey + f"\\{key_name}"
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4220/commits/342572271a42610bdc01315915bc2da74fdb1a83